internal/test_unit/xml/unit_tc_pairs.xml

```diff
@@ -206,4 +206,27 @@
     </output>
   </test>
 
+  <test name="tc_pairs_DIAGNOSTICS_CONSENSUS">
```

Comment:
Note that this remains not a very good test. Recommend replacing this with a more realistic test of the functionality in future work.